src/statement.js

```diff
@@ -0,0 +1,22 @@
+function Statement(transactions) {
```

Comment:
Great that you broke out the statement printing responsibility into a separate object!